Libraries/LLM/Evaluate.swift

```diff
@@ -5,56 +5,12 @@ import MLX
 import MLXRandom
 import Tokenizers
 
-private func topPSampling(logits: MLXArray, topP: Float, temp: Float) -> MLXArray {
```

Comment:
These functions move inside the SampleContext for encapsulation.  They also get compiled to match the python side.